### PR TITLE
added support for people and organization

### PR DESCRIPTION
**Pull request by sprajjwal**
same as topic
Screen Shot 2020-06-04 at 7 35 21 PM

### CODE REVIEW

File: src/AppItemDetails.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const ReactMarkdown = require('react-markdown')
 
 class AppItemDetails extends React.Component {
   render() {
+    console.log(this.props)
```
Comment (ytsheng) [inline src/AppItemDetails.js:8]: remove logging

File: src/AppItemDetails.js [hunk 2]
```diff
@@ -18,6 +19,10 @@ ${this.props.country_coverage}
 ${this.props.state_adoption}
 #### Target User Type
 ${this.props.target_audience}
+#### Creators:
```
Comment (ytsheng) [inline src/AppItemDetails.js:22]: So, remove the colon there... Should be consistent from above

Comment (sprajjwal) [reply src/AppItemDetails.js:22]: oops

File: src/AppItemDetails.js (hunk 1 above)
Comment (sprajjwal) [reply src/AppItemDetails.js:8]: that shouldn't be there

**Comment by ytsheng** [comment]
This is looking good!!! Thank you so much!

**Comment by ytsheng** [comment]
Waiting for another PR for the data population before landing this one.

**Pull request merged**